hc/front/tests/test_my_checks.py

```diff
@@ -58,3 +58,19 @@ def test_it_shows_amber_check(self):
 
         # Mobile
         self.assertContains(r, "label-warning")
```

Comment:
Good tests
Good colour options for the often icon
Good and comprehensive pull request